src/config/HealthConfig.ts

```diff
@@ -8,18 +8,19 @@ export const parseHealthConfig = (healthConfig: HealthConfig) => {
     healthConfig.rate[key].name = getExpr(healthConfig.rate[key].name);
     healthConfig.rate[key].kind = getExpr(healthConfig.rate[key].kind);
     for (let t of Object.values(r.tolerance)) {
-      t.code = getExpr(t.code);
+      t.code = getExpr(t.code, true);
       t.direction = getExpr(t.direction);
       t.protocol = getExpr(t.protocol);
     }
   }
   return healthConfig;
 };
 
-export const getExpr = (value: RegexConfig | undefined): RegExp => {
+export const getExpr = (value: RegexConfig | undefined, code: boolean = false): RegExp => {
```

Comment:
I'm thinking if I need to maintain this function, I'd appreciate a litle bit of more context.

Suggestion:
```
export const getExpr = (value: RegexConfig | undefined, code: boolean = false)
```
->
```
export const getHealthRegExp = (value: RegexConfig | undefined, isCode: boolean = false)
```

Not a blocker, but it helped me to follow the UI changes.